docs/advanced/api-server-proxy.md

```diff
@@ -0,0 +1,23 @@
+# API Server Proxy
```

Comment:
@aleoli This documentation looks nice, but it's not clear where this has to be set up (in the cluster that export the resources? IN the other?) I feel this should be clarified in the documentation.
In addition, it's not clear to me which is the IP address I would use, in my cluster, to connect to the remote API server.
Finally, this feature has nothing to do with the flag that enables an application to contact either its local or its remote API server, isn't it?

Reply:
Users do not need to set up anything; this is done by default by Liqo at installation time. This page describes the functionality.

No, there are different things. This allows network interconnection, while the flag specifies which variable to inject in the pod to contact the local or remote API server. Of course, if it needs to contact the one in the other cluster, it will use the mapping done here.